main.py

```diff
@@ -121,66 +126,60 @@ def update_mw(self, request: Request) -> JSONResponse:
         mw_id: MaintenanceID = request.path_params["mw_id"]
         old_maintenance = self.scheduler.get_maintenance(mw_id)
         if old_maintenance is None:
-            raise HTTPException(
-                404, detail=f'Maintenance with id {mw_id} not found'
-            )
+            raise HTTPException(404, detail=f"Maintenance with id {mw_id} not found")
         if old_maintenance.status == Status.RUNNING:
             raise HTTPException(
-                400, detail='Updating a running maintenance is not allowed'
+                400, detail="Updating a running maintenance is not allowed"
             )
-        if 'status' in data:
+        if "status" in data:
             raise HTTPException(
-                400, detail='Updating a maintenance status is not allowed'
+                400, detail="Updating a maintenance status is not allowed"
             )
         try:
             new_maintenance = MW.parse_obj({**old_maintenance.dict(), **data})
         except ValidationError as err:
             msg = error_msg(err.errors())
             raise HTTPException(400, detail=msg) from err
         if new_maintenance.id != old_maintenance.id:
-            raise HTTPException(400, detail='Updated id must match old id')
+            raise HTTPException(400, detail="Updated id must match old id")
         self.scheduler.update(new_maintenance)
-        return JSONResponse({'response': f'Maintenance {mw_id} updated'})
+        return JSONResponse({"response": f"Maintenance {mw_id} updated"})
 
-    @rest('/v1/{mw_id}', methods=['DELETE'])
+    @rest("/v1/{mw_id}", methods=["DELETE"])
     def remove_mw(self, request: Request) -> JSONResponse:
         """Delete a maintenance window."""
         mw_id: MaintenanceID = request.path_params["mw_id"]
         maintenance = self.scheduler.get_maintenance(mw_id)
         if maintenance is None:
-            raise HTTPException(
-                404, detail=f'Maintenance with id {mw_id} not found'
-            )
+            raise HTTPException(404, detail=f"Maintenance with id {mw_id} not found")
         if maintenance.status == Status.RUNNING:
             raise HTTPException(
-                400, detail='Deleting a running maintenance is not allowed'
+                400, detail="Deleting a running maintenance is not allowed"
             )
         self.scheduler.remove(mw_id)
-        return JSONResponse({'response': f'Maintenance with id {mw_id} '
-                                         f'successfully removed'})
+        return JSONResponse(
+            {"response": f"Maintenance with id {mw_id} " f"successfully removed"}
```

Comment:
Should probably remove the string concatenation, and on top of that the string on the rhs should probably not even be an f-string as it doesn't even format the string.

Reply:
I just unconcatenated the string.